dataverse-webapp/src/main/webapp/metrics.xhtml

```diff
@@ -110,12 +114,31 @@
                              type="bar"
                              model="#{UsersChart.usersChart}"
                              style="height:300px"
-                             responsive="true" />
+                             responsive="true"
+                    />
                 </p:panelGrid>
             </h:form>
+
+            <!-- Used in ChartCreator class as extender -->
+            <script type="text/javascript">
+                function customizeChart()
```

Comment:
Don't know why, but when you switch to month chart then chart title is showing up. When we switch to year it is gone

Reply:
It was a bug on in my code.
What caught my attention was xAxis overlap on small resolutions. 
I've added an angle for label display but which fixes the problem but looks rather dreadful. 
I guess it must be consulted with @wfenrich